### DISCUSSION TITLE
Does it load the card in the background while it doesn't ?

### DISCUSSION THREAD

**Discussion by MrDrew514** [kinghat/tabbed-card#144] [Q&A]
Hello, I'm using this card to show a couple of cameras in different tabs but it seems like it's loading up everyhing at the same time. Any way to render the current tab only? thanks again

**Comment by kinghat** [comment]
> Hello, I'm using this card to show a couple of cameras in different tabs but it seems like it's loading up everyhing at the same time. Any way to render the current tab only? thanks again

what makes it seem like its loading everything up?

**Reply by MrDrew514** [reply]
It's acting diffrently when I change tabs using the card and when I change to another view or page. Is it mean to not load the cards in the background? If so I'll look into my camera card configuration,

**Reply by kinghat** [reply]
iirc the tab wont be loaded until you view it. then its in memory. it will be unloaded if you navigate to a different dashboard.

**Reply by MrDrew514** [reply]
Thank you for the answer! Is it the same result as a conditional card? I'd need to have the card unload on tab change

**Reply by kinghat** [reply]
i honestly have no idea :man_shrugging:
